infra/modules/ioweb_profile_api/api/bff/policy.xml

```diff
@@ -0,0 +1,31 @@
+<policies>
+  <inbound>
+    <base />
+    <set-header name="x-functions-key" exists-action="override">
+      <value>{{ioweb-profile-api-key}}</value>
+    </set-header>
+    <cors>
+      <allowed-origins>
+        <origin>https://ioapp.it/</origin>
+      </allowed-origins>
```

Comment:
We can apply the new domain as well.
PR https://github.com/pagopa/io-infra/pull/1337 is not needed anymore.
cc @arcogabbo

Reply:
fixed in [fc8df85](https://github.com/pagopa/io-auth-n-identity-domain/pull/129/commits/fc8df856eb79256cd9140b63111f285900f15245)